build.sh

```diff
@@ -1,39 +1,45 @@
 #!/bin/bash
+BGREEN='\033[1;32m'
+GREEN='\033[0;32m'
+BCYAN='\033[1;36m'
+RED='\033[0;31m'
+
+NC='\033[0m' # Removes Color
+
 baseOutDir=$1
 if [ -n "$baseOutDir" ]; then
-	if [ ! -d "$baseOutDir" ]; then
-		echo "Making base output directory $baseOutDir"
-		mkdir "$baseOutDir"
-	fi
+        if [ ! -d "$baseOutDir" ]; then
+                echo -e "${GREEN}Making base output directory ${BGREEN}$baseOutDir\n"
+                mkdir "$baseOutDir"
+        fi
 fi
 
 build_dir () {
-	srcDir=$1
-	dstDir=$2
-	if [ -n "$baseOutDir" ]; then
-		dstDir="$baseOutDir/$dstDir"
-	fi
+        srcDir=$1
+        dstDir=$2
+        if [ -n "$baseOutDir" ]; then
+                dstDir="$baseOutDir/$dstDir"
+        fi
+        params=( "$@" )
+        rest=( "${params[@]:2}" )
 
-	params=( "$@" )
-	rest=( "${params[@]:2}" )
+        if [ ! -d "$dstDir" ]; then
+                mkdir -p "$dstDir"
+        else
+                rm "$dstDir"/*.txt
+        fi
 
-	if [ ! -d "$dstDir" ]; then
-		echo "Making output directory $dstDir"
-		mkdir -p "$dstDir"
-	else
-		rm "$dstDir"/*.txt
-	fi
+        for f in "$srcDir"/*.txt
+        do
+                filename=`basename $f`
+                outName="$dstDir/$filename"
+                #echo "Building $f to $outName with params ${rest[@]}"
```

Comment:
echo -e "${GREEN}[PHP] ${NC}Exec $f to $outName with params ${rest[@]}"

the php invocations are the main build step, why silence those